hawc/apps/common/management/commands/dump_test_db.py

```diff
@@ -11,7 +11,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
-        if "test" not in settings.DATABASES["default"]["NAME"]:
+        if not any(_ in settings.DATABASES["default"]["NAME"] for _ in ["fixture", "test"]):
```

Comment:
do we need test here? I think if we're suggesting we use hawc-fixture for modifications, we might just need that